internal/pkg/util/configurebccsp_test.go

```diff
@@ -0,0 +1,61 @@
+// +build pkcs11
```

Comment:
Still not sure any tests are run with this tag.

Reply:
Agreed, I ran the test manually.